packages/@atjson/source-prism/src/annotations/media.ts

```diff
@@ -0,0 +1,6 @@
+import { BlockAnnotation } from '@atjson/document';
+
+export default class Media extends BlockAnnotation {
+  static vendorPrefix = 'pam';
```

Comment:
What is this vendor prefix?

Reply:
Oh, it's the xml namespace

Reply:
The Prism standard includes tags from a few standard namespaces, which are specified in the `xmlns` declarations on the root `pam:message` node:

pam: http://prismstandard.org/namespaces/pam/2.0/
dc: http://purl.org/dc/elements/1.1/
dcterms: http://purl.org/dc/terms/
xhtml: http://www.w3.org/1999/xhtml 
pim: http://prismstandard.org/namespaces/pim/2.0/ 
prism: http://prismstandard.org/namespaces/basic/2.0/ 
prl http://prismstandard.org/namespaces/prl/2.0/
xsi: http://www.w3.org/2001/XMLSchema-instance

Where we find tags under those namespaces, we use the namespace as the vendor prefix.

Reply:
Thanks for the thorough explanation!